sdf/mandelbulbSDF.glsl

```diff
@@ -3,7 +3,11 @@
 /*
 original_author: Kathy McGuiness
 description: Returns the mandelbulb SDF
+For more information about the mandlebulb, check [this article](https://en.wikipedia.org/wiki/Mandelbulb)
+
 use: mandelbulbSDF(<vec2> st)
+Example:
+https://gist.github.com/kfahn22/4c29e6d2bdc33d639f315edaa934d287
```

Comment:
Same here, respect the YAML structure of the comment header. `examples` field is not capitalize and examples are part of of a list.
```yaml
examples:
    - https://gist.githubusercontent.com/kfahn22/4c29e6d2bdc33d639f315edaa934d287/raw/0eb94683614995476fb63cfa99881e420f1c5be6/mandelbulb.frag
```

also the example needs to be a text page not html. Use the RAW link

Reply:
I have updated the file.